privatebin/values.yaml

```
@@ -15,13 +15,19 @@ image:
 
 nameOverride: ""
 fullnameOverride: ""
+datapath: "/srv/data"
 
 service:
   type: ClusterIP
   port: 80
   portName: http
   annotations: {}
 
+controller:
+  kind: "Deployment"
+  pvc:
+    requests: "1Gi"
```

This is to keep consistent with how other charts manage persistent volumes.

```suggestion
    requests: "1Gi"
    ## logstash data Persistent Volume Storage Class
    ## If defined, storageClassName: <storageClass>
    ## If set to "-", storageClassName: "", which disables dynamic provisioning
    ## If undefined (the default) or set to null, no storageClassName spec is
    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
    ##   GKE, AWS & OpenStack)
    ##
    # storageClass: "-"    
```